### DISCUSSION TITLE
Handling for Linkdown Event in vSomeIP Implementation

### DISCUSSION THREAD

**Discussion by zhaoxian1989** [COVESA/vsomeip#702] [Q&A]
https://github.com/COVESA/vsomeip/blob/master/implementation/endpoints/src/netlink_connector.cpp
According to the SOME/IP protocol, it is recommended to clear out any registered subscriptions upon detecting a linkdown event. However, upon reviewing the vSomeIP implementation, it appears that there is no specific handling for the linkdown event.

Question:
Could you please clarify why the vSomeIP implementation does not include handling for the linkdown event as per the SOME/IP protocol recommendation? Is there a specific reason for this omission, or is it planned to be addressed in future updates?

**Comment by duartenfonseca** [comment]
@zhaoxian1989 could you create an issue with this problem, so that we can reproduce it and try to help further?
